benchexec/tools/abc.py

```diff
@@ -24,7 +25,33 @@ def name(self):
         return "ABC"
 
     def cmdline(self, executable, options, task, rlimits):
-        return [executable] + options + [task.single_input_file]
+        return (
+            [executable]
+            + ["-c", "&r {}; &put".format(task.single_input_file)]
+            + options
+        )
+
+    def determine_result(self, run):
+        """
+        @return: status of ABC after executing a run
+        """
+        if run.was_timeout:
+            return "TIMEOUT"
```

Comment:
Please use `result.RESULT_TIMEOUT` instead of a literal.

But most of the time, this check is not actually needed because BenchExec sets the timeout result anyway. Only if you want to avoid returning any other irrelevant result below you need this.

Same for avr.py, of course.

Reply:
I cannot find the definition of `RESULT_TIMEOUT` in `result.py`.

Reply:
Ahh, it is added just recently. I will merge the new changes first.

Reply:
Addressed in 0ef3c7f